lib/linguist/languages.yml

```diff
@@ -2074,14 +2074,11 @@ HTML+Django:
   - ".jinja"
   - ".j2"
   - ".jinja2"
-  - ".njk"
   aliases:
   - django
   - html+django/jinja
   - html+jinja
   - htmldjango
-  - njk
-  - nunjucks
```

Comment:
Don't lose me in the trip down south 😁

Reply:
You've still forgotten to include me in the new section 😉

Reply:
Which bit? line 2084? Aren't lowercase forms already automatically aliases?

Reply:
🤦 I forgot about that. Ignore me 😊